### PR TITLE
Tweak sizing and spacing and remove redundant alert.

### PR DESCRIPTION
**Pull request by dcndr**
(removed alert because feature not gonna be part of prod. release)

### CODE REVIEW

File: frontend/src/components/Header/index.tsx
Comment (circles-png) [inline frontend/src/components/Header/index.tsx:1]: on mobile the pencil icon is gone
always develop mobile first

**Review by circles-png — CHANGES_REQUESTED** [review]
pencil icon in header is gone at lower screen sizes with a long school name

File: frontend/src/components/Header/index.tsx [hunk 1]
```diff
@@ -25,15 +25,15 @@ const Header = () => {
...
+      <h1 className="md:text-xl">Welcome to</h1>
       <span className={classNames(
         "rounded-xl pl-1 w-full text-xl font-bold flex gap-x-4 items-center transition",
         focused ? 'bg-gray-200' : ''
       )}>
         <input
           type='text'
           placeholder="School name"
-          className="bg-transparent w-full text-center text-3xl"
+          className="bg-transparent w-full text-center md:text-3xl sm:text:xl"
```
Comment (circles-png) [inline frontend/src/components/Header/index.tsx:36]: to make a class apply only on one screen size do `default-class md:class-that-applies-only-on-at-least-md`
`md:` means this class applies only on medium and above

also `text:xl` doesnt mean anything

should probably be `text-xl md:text-3xl`

**Review by circles-png — CHANGES_REQUESTED** [review]

File: frontend/src/components/Header/index.tsx (hunk 1 above)
Comment (dcndr) [reply frontend/src/components/Header/index.tsx:36]: ma bad lmfao

Comment (dcndr) [reply frontend/src/components/Header/index.tsx:36]: amended

**Comment by circles-png** [comment]
image
right?

**Comment by circles-png** [comment]
or maybe solid bg

**Review by circles-png — APPROVED** [review]

**Review by circles-png — APPROVED** [review]

**Pull request merged**